src/language_features/completion.rs

```diff
@@ -66,9 +66,9 @@ pub fn editor_completion(
             }
             format!(
                 "{}|{}|{}",
-                re.replace_all(&x.insert_text.unwrap_or(x.label), r"\$c"),
+                x.insert_text.unwrap_or(x.label),
```

Comment:
I think we still want to escape `:` and `|` because they are delimiters in completions list (see https://github.com/mawww/kakoune/blob/master/doc/interfacing.asciidoc#completion-candidates). Let's just remove `$` from regex on line 44. I'm not sure why it's there, looks like I copy-pasted it from somewhere though from quick look to Kakoune sources `$` doesn't have any special meaning in completions list.

Reply:
I removed the $ from the regex.